### ISSUE TITLE
Add a default scheduled job

### ISSUE THREAD

**Issue by ergonlogic** [ginkgostreet/civicrm-metrics-client#5]
In order for the 'collate' action to run, a custom Job needs to be defined. I believe that this could be automated as part of installing this extension via [hook_civicrm_managed](https://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_managed)

**Comment by caracolmzd** [comment]
I agree, this would help get the extension ready for "prime time". It is not completely accurate that a scheduled job is required to collate the metrics, what is requires is invoking the hook, either by custom code or through the API call. 
I can see why we did not prioritize this, as creating a scheduled job to call the api through the interface is trivial. Perhaps the documentation could steer users in that direction better.

**Comment by caracolmzd** [comment]
Resolved by PR #6

**Comment by caracolmzd** [comment]
whoops, merged too soon.
